Vagrantfile

```diff
@@ -7,7 +7,7 @@ sudo apt-get install -y openjdk-7-jdk git
 SCRIPT
 
 $script = <<SCRIPT
-VERSION=1.565
+VERSION=1.580
```

Comment:
It's recommended for plugins to depend on the base version of a LTS release line. I think I saw that recommendation on the Jenkins wiki, but I can't find it now.